### ISSUE TITLE
Logs from docker compose are not update in 0.11.0 and still mention the old ports

### ISSUE THREAD

**Issue by arcones** [localstack/localstack#2291]
# Type of request: This is a bug report

# Detailed description
Logs of localstack docker image 0.11.0, when used with docker-compose, are still referring to old ports without any warning about its deprecation. This could be misleading as, even when the edge router 4566 is mentioned, there logs don't tell the recommended ports to use, and which are deprecated.

## Expected behavior
localstack    | Starting edge router (http port 4566)...
localstack    | Starting edge router (https port 443)...
localstack    | Starting mock Lambda service (http port 4574 (deprecated) and 4566 (recommended))...
localstack    | Starting mock CloudWatch Logs  (http port 4568 (deprecated) and 4566 (recommended))...
localstack    | Waiting for all LocalStack services to be ready
localstack    | Starting mock SNS  (http port 4575 (deprecated) and 4566 (recommended))...

## Actual behavior
localstack    | Starting edge router (http port 4566)...
localstack    | Starting edge router (https port 443)...
localstack    | Starting mock Lambda service (http port 4574)...
localstack    | Starting mock CloudWatch Logs (http port 4586)...
localstack    | Waiting for all LocalStack services to be ready
localstack    | Starting mock SNS (http port 4575)...

# Steps to reproduce
Just run `docker-compose up` and see the logs

## Command used to start LocalStack
`docker-compose up`

**Comment by whummer** [comment]
Good point, thanks for reporting @arcones . Would you be able to create a small pull request to fix the log output with the suggested changes? Thanks!

**Comment by arcones** [comment]
Sure, I am happy to contribute!!!

**Comment by arcones** [comment]
@whummer could you give me permission to upload my branch in order to create a pull request?
currently I get:
```
~/Projects/Personal/localstack(2291-logs-bug) » gpsh -u origin $(git branch | sed -n -e 's/^\* \(.*\)/\1/p')
ERROR: Permission to localstack/localstack.git denied to arcones.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

**Comment by whummer** [comment]
Thanks @arcones . The way this works on Github is that you can create a *fork* of the repository into your own user account.

The fork would get created into https://github.com/arcones/localstack . You can then clone the code from your fork of the repo, and push your changes to your fork. Once the changes have been pushed to your fork, we can create a pull request against this repository.

Hope that helps. Please let me know if anything is unclear.

**Comment by whummer** [comment]
Fixed in #2318 - thanks again. 👍
